checkpointctl.go

```diff
@@ -95,6 +96,12 @@ func setupInspect() *cobra.Command {
 		false,
 		"Display an overview of processes in the container checkpoint",
 	)
+	flags.BoolVar(
+		&psTreeCmd,
+		"ps-tree-cmd",
```

Comment:
Would it make more sense to name this something like `ps-args`? It's more intuitive, going by the description.

Reply:
Hum, I prefer `ps-tree-cmd` or maybe `ps-tree-args` ? Because it is still the the process tree that we are dealing with.

Reply:
`cmd` is what the `ps` utility is using (e.g., `ps a -o pid,cmd`) and I suspect that most users would already be familiar with this tool. Therefore, it might be more familiar for them than `args`. 

IMHO, we currently have the `--ps-tree` option and by concatenating `-cmd` it would more obvious to users that it is an alternative option with extended functionality, and autocomplete would work better ;)

For example:
```
./checkpointctl inspect --ps-tree /tmp/checkpoint.tar.gz
```
vs
```
./checkpointctl inspect --ps-tree-cmd /tmp/checkpoint.tar.gz
```

Reply:
Hmm, makes sense